src/acceptance/assets/app/go_app/internal/generate.go

```diff
@@ -1,3 +1,6 @@
 package internal
 
-//go:generate go run github.com/ogen-go/ogen/cmd/ogen --no-server --debug.ignoreNotImplemented "mutualTLS security" --target custommetrics --clean ../../../../../../api/custom-metrics-api.yaml
+//go:generate go run github.com/ogen-go/ogen/cmd/ogen --no-server --debug.ignoreNotImplemented "mutualTLS security" --target applicationmetric --clean ../internal/openapi-specs.bundled/application-metric-api.openapi.bundled.yaml
+//go:generate go run github.com/ogen-go/ogen/cmd/ogen --no-server --debug.ignoreNotImplemented "mutualTLS security" --target custommetrics --clean ../internal/openapi-specs.bundled/custom-metrics-api.openapi.bundled.yaml
+//go:generate go run github.com/ogen-go/ogen/cmd/ogen --no-server --debug.ignoreNotImplemented "mutualTLS security" --target policy --clean ../internal/openapi-specs.bundled/policy-api.openapi.bundled.yaml
+//go:generate go run github.com/ogen-go/ogen/cmd/ogen --no-server --debug.ignoreNotImplemented "mutualTLS security" --target scalinghistory --clean ../internal/openapi-specs.bundled/scaling-history-api.openapi.bundled.yaml
```

Comment:
'-debug.ignoreNotImplemented "mutualTLS security"` should only be needed for custom metrics API, as it is the only providing this security scheme.

```suggestion
//go:generate go run github.com/ogen-go/ogen/cmd/ogen --no-server --target applicationmetric --clean ../internal/openapi-specs.bundled/application-metric-api.openapi.bundled.yaml
//go:generate go run github.com/ogen-go/ogen/cmd/ogen --no-server --debug.ignoreNotImplemented "mutualTLS security" --target custommetrics --clean ../internal/openapi-specs.bundled/custom-metrics-api.openapi.bundled.yaml
//go:generate go run github.com/ogen-go/ogen/cmd/ogen --no-server --target policy --clean ../internal/openapi-specs.bundled/policy-api.openapi.bundled.yaml
//go:generate go run github.com/ogen-go/ogen/cmd/ogen --no-server --target scalinghistory --clean ../internal/openapi-specs.bundled/scaling-history-api.openapi.bundled.yaml
```

Reply:
Add `--allow-remote` to be able to work on the unprocessed files